website/en/docs/editors/vim.md

```diff
@@ -2,28 +2,52 @@
 layout: guide
 ---
 
+### ALE <a class="toc" id="vim-ale" href="#vim-ale"></a>
+
+The Asynchronous Lint Engine (ALE) [vim-ale](https://github.com/w0rp/ale) plugin for Vim 8+ and NeoVim provides linting of many syntaxes during editing, before files are saved. Supported JavaScript linters include eslint, jscs, jshint, flow, standard, and xo.  Installation instructions can be found at https://github.com/w0rp/ale#3-installation.
+
+Here is suggested ALE configuration to place in your .vimrc file:
+```
+" Asynchronous Lint Engine (ALE)
+" Limit linters used for JavaScript.
+let g:ale_linters = {
+\  'javascript': ['eslint', 'flow'],
+\  'scss': ['stylelint']
+\}
+highlight clear ALEErrorSign " otherwise uses error bg color (typically red)
+highlight clear ALEWarningSign " otherwise uses error bg color (typically red)
+let g:ale_sign_error = '👎'
```

Comment:
Could you escape these emoji? Looks like this should work: https://stackoverflow.com/questions/34538879/unicode-in-github-markdown

Some of our infrastructure will still complain if we commit literal emojis.